interop/configs/commit.json

```diff
@@ -0,0 +1,128 @@
+{
+  "scripts": {
+    "empty": [
+      {"action": "createGroup", "actor": "alice"},
+      {"action": "externalJoin", "actor": "alice", "joiner": "bob"},
+      {"action": "fullCommit", "actor": "alice", "members": ["bob"]}
+    ],
+
+    "add": [
+      {"action": "createGroup", "actor": "alice"},
+      {"action": "externalJoin", "actor": "alice", "joiner": "bob"},
+
+      {"action": "createKeyPackage", "actor": "charlie1"},
+      {"action": "createKeyPackage", "actor": "charlie2"},
+      {"action": "addProposal", "actor": "alice", "keyPackage": 2},
+      {"action": "addProposal", "actor": "bob", "keyPackage": 3},
+      {"action": "fullCommit", "actor": "alice", "byReference": [4, 5], "members": ["bob"], "joiners": ["charlie1", "charlie2"], "force_path": true},
+
+      {"action": "createKeyPackage", "actor": "charlie3"},
+      {"action": "createKeyPackage", "actor": "charlie4"},
+      {"action": "addProposal", "actor": "alice", "keyPackage": 7},
+      {"action": "addProposal", "actor": "bob", "keyPackage": 8},
+      {"action": "fullCommit", "actor": "alice", "byReference": [9, 10], "members": ["bob", "charlie1", "charlie2"], "joiners": ["charlie3", "charlie4"], "force_path": false}
+    ],
+
+    "remove": [
+      {"action": "createGroup", "actor": "alice"},
+      {"action": "externalJoin", "actor": "alice", "joiner": "bob"},
+      {"action": "createKeyPackage", "actor": "charlie1"},
+      {"action": "createKeyPackage", "actor": "charlie2"},
+      {"action": "addProposal", "actor": "alice", "keyPackage": 2},
+      {"action": "addProposal", "actor": "alice", "keyPackage": 3},
+      {"action": "fullCommit", "actor": "alice", "byReference": [4, 5], "members": ["bob"], "joiners": ["charlie1", "charlie2"]},
```

Comment:
Is there a reason to do this with `addProposal` and not `externalJoin`?

Reply:
Also, let's pick a single naming scheme here, either alphabetical (`alice`, `bob`, `charlie`, ...) or numeric (`alice1`, `alice2`, `alice3`, ...)

Reply:
I forgot to update this one. Should be better now.